docs/user/vsphere/privileges.md

```diff
@@ -35,28 +35,32 @@ The easiest way to ensure proper permissions is to grant Global Permissions to t
 
 For more information, consult [vSphere Permissions and User Management Tasks][vsphere-perms]
 
-### Precreated virtual machine folder
+### Precreated virtual machine folder in vSphere vCenter
 
 Role Name | Propagate | Entity
 --- | --- | ---
-openshift-vcenter-level | False | vSphere vCenter
-ReadOnly | False | vSphere vCenter Datacenter
-openshift-cluster-level | True | vSphere vCenter Cluster
-openshift-datastore-level | False | vSphere vCenter Datastore
-ReadOnly | False | vSphere Switch
-openshift-portgroup-level | False | vSphere Port Group
-openshift-folder-level | True | vSphere vCenter Virtual Machine folder
-
-
-### Installer created virtual machine folder
+openshift-vcenter-level | False | vCenter
+ReadOnly | False | Datacenter
+openshift-cluster-level | True | Cluster
+openshift-datastore-level | False | Datastore
+ReadOnly | False | Switch
+openshift-portgroup-level | False | Port Group
+ReadOnly | True | Virtual Machine folder (Top Level)
+openshift-folder-level | True | Virtual Machine folder
+
+In a cascading folder organization you will need  "`Read-only`" permissions with "`Propagate to children`" from the top folder level.
```

Comment:
@codekow not quite sure `Virtual Machine folder (Top Level)` is trying to convey. Do you mean if I have a nested folder structure the top-level of those folders? 

The _real_ top level `/datacenter/vm` is not displayed within vCenter.

Reply:
Yes, specifically I am talking about a nested folder structure. This is specific to the user view and not the internal view. @jcpowermac